### PR TITLE
dx-1157 Cleaning up the test data a bit.

### PR DESCRIPTION
**Pull request by arebra**
Did some language clean-up and tried to remove information that's not of use to a costumer. Also added 3DS emulator screenshots.

### CODE REVIEW

File: resources/test-data.md [hunk 1]
```diff
@@ -260,3 +264,5 @@ To simulate an error message, set description in `POST` Create Payment or
 | BANKIDCL    | Payer cancelled BankId signing                 |
 
 [create-card-purchase]: /payments/card/redirect#step-1-create-a-purchase
+[3DS-emulator-no-dropdown]: /assets/img/3DS-emulator-no-dropdown.png
+[3DS-emulator-with-dropdown]: /assets/img/3DS-emulator-with-dropdown.png
```
Comment (payex-dev) [inline resources/test-data.md:268]: Codacy found an issue: [[definition-case] Do not use uppercase characters in definition labels](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6100690)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6100690&bid=19970713)

File: resources/test-data.md [hunk 2]
```diff
@@ -260,3 +264,5 @@ To simulate an error message, set description in `POST` Create Payment or
 | BANKIDCL    | Payer cancelled BankId signing                 |
 
 [create-card-purchase]: /payments/card/redirect#step-1-create-a-purchase
+[3DS-emulator-no-dropdown]: /assets/img/3DS-emulator-no-dropdown.png
```
Comment (payex-dev) [inline resources/test-data.md:267]: Codacy found an issue: [[definition-case] Do not use uppercase characters in definition labels](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6100690)

File: resources/test-data.md [hunk 3]
```diff
@@ -143,36 +143,38 @@ amounts set to trigger errors in our test environment.
...
+| Card type  | Card number        | Expiry | CVC   | Type of test data   |
+| :--------- | :----------------- | :----- | :---- | :------------------ |
+| Visa       | `4761739001010416` | 12/22  | `268` | 3-D Secure enrolled |
+| MasterCard | `5226612199533406` | 09/28  | `602` | 3-D Secure enrolled |
 
 After pressing the purchase button you will then be taken to a menu where you
 can select Authentication status.
 
-<!--- TODO: REMEMBER TO ADD PICTURE HERE -->
+![3D-Secure Emulator without dropdown menu][3DS-emulator-no-dropdown]{:width="805px" :height="685px"}
```
Comment (asbjornu) [inline resources/test-data.md:157]:
```suggestion
![3D-Secure Emulator without dropdown menu][3ds-emulator-no-dropdown]{:width="805px" :height="685px"}
```

File: resources/test-data.md [hunk 4]
```diff
@@ -143,36 +143,38 @@ amounts set to trigger errors in our test environment.
...
 
-In this menu there is a few different options to choose from, choose the status
+![3D-Secure Emulator with dropdown menu][3DS-emulator-with-dropdown]{:width="805px" :height="685px"}
```
Comment (asbjornu) [inline resources/test-data.md:159]:
```suggestion
![3D-Secure Emulator with dropdown menu][3ds-emulator-with-dropdown]{:width="805px" :height="685px"}
```

File: resources/test-data.md (hunk 2 above)
Comment (asbjornu) [inline resources/test-data.md:267]:
```suggestion
[3ds-emulator-no-dropdown]: /assets/img/3DS-emulator-no-dropdown.png
```

File: resources/test-data.md (hunk 1 above)
Comment (asbjornu) [inline resources/test-data.md:268]:
```suggestion
[3ds-emulator-with-dropdown]: /assets/img/3DS-emulator-with-dropdown.png
```

File: resources/test-data.md [hunk 5]
```diff
@@ -225,11 +227,12 @@ Use any name, address etc.
...
 ## Vipps Test Data
 
-For testing a positive purchase (in our external integration test environment),
-please use any mobile number, except within the range: 99999991-99999999, as
-these will trigger error messages. The error messages are documented in the
-table below. There will be no user dialog at the mobile phone when testing
-Vipps.
+Testing a successful Vipps purchase (in our external integration test 
+environment) can be done using any valid Norwegian mobile number, E.g: 
++47 99999999 except within the range: **9999991-99999998**, as these will
```
Comment (asbjornu) [inline resources/test-data.md:232]:
```suggestion
`+47 99999999` except within the range: `9999991` - `99999998`, as these will 
```

File: resources/test-data.md [hunk 6]
```diff
@@ -246,11 +249,12 @@ Vipps.
...
 ## Swish Test Data
 
-For testing a positive purchase (in our external integration test environment),
-please use any mobile number. E.g: +46 739000001
-
-To simulate an error message, set description in `POST` Create Payment or
-`Create` Payment Order to one of the following values:
+Testing a successful Swish purchase (in our external integration test 
+environment) can be done by using any valid Swedish mobile number. E.g: 
++46 739000001. As with Vipps, the external integration test environment uses a
```
Comment (asbjornu) [inline resources/test-data.md:254]:
```suggestion
`+46 739000001`. As with Vipps, the external integration test environment uses a 
```

**Review by asbjornu — CHANGES_REQUESTED** [review]
👍

**Review by asbjornu — APPROVED** [review]

**Pull request merged**